integration_tests/conftest.py

```diff
@@ -20,14 +20,17 @@ def sim() -> InteractiveContext:
 
 
 @pytest.fixture(scope="session")
-def tracked_live_populations(sim) -> Dict[int, pd.DataFrame]:
-    previous_step_number = 0
-    population_states = {}
-    for step in TIME_STEPS_TO_TEST:
-        sim.take_steps(step - previous_step_number)
-
+def populations(sim) -> List[pd.DataFrame]:
+    population_states = []
+    for _ in range(max(TIME_STEPS_TO_TEST) + 1):
         pop = sim.get_population()
```

Comment:
Note this is not going to return untracked simulants. Consider adding the `untracked=True` argument to `get_population` and filtering out untracked in addition to dead in the `tracked_live_populations` fixture below.

Reply:
I've just added this -- I will need it for emigration.